pkg/preflight/assets/host-preflights.yaml

```
@@ -140,6 +152,63 @@ spec:
               message: The disk containing directory /var/lib/docker has less than 10Gi of disk space available
           - pass:
               message: The disk containing directory /var/lib/docker has at least 30Gi of total space, has at least 10Gi of disk space available, and is less than 60% full.
+    - diskUsage:
+        checkName: "Ephemeral Disk Usage /var/lib/containerd"
+        collectorName: "Ephemeral Disk Usage /var/lib/containerd"
+        exclude: '{{kurl not .Installer.Spec.Containerd.Version}}'
+        outcomes:
+          - fail:
+              when: "total < 30Gi"
+              message: The disk containing directory /var/lib/containerd has less than 30Gi of total space
+          - fail:
+              when: "used/total > 80%"
+              message: The disk containing directory /var/lib/containerd is more than 80% full
+          - warn:
+              when: "used/total > 60%"
+              message: The disk containing directory /var/lib/containerd is more than 60% full
+          - warn:
+              when: "available < 10Gi"
+              message: The disk containing directory /var/lib/containerd has less than 10Gi of disk space available
+          - pass:
+              message: The disk containing directory /var/lib/containerd has at least 30Gi of total space, has at least 10Gi of disk space available, and is less than 60% full.
+    - diskUsage:
+        checkName: "Ephemeral Disk Usage /var/lib/rook"
+        collectorName: "Ephemeral Disk Usage /var/lib/rook"
+        exclude: '{{kurl not .Installer.Spec.Containerd.Version}}'
+        outcomes:
+          - fail:
+              when: "total < 10Gi"
+              message: The disk containing directory /var/lib/rook has less than 10Gi of total space
+          - fail:
+              when: "used/total > 80%"
+              message: The disk containing directory /var/lib/rook is more than 80% full
+          - warn:
+              when: "used/total > 60%"
+              message: The disk containing directory /var/lib/rook is more than 60% full
+          - warn:
+              when: "available < 10Gi"
+              message: The disk containing directory /var/lib/rook has less than 10Gi of disk space available
+          - pass:
+              message: The disk containing directory /var/lib/rook has at least 10Gi of total space, has at least 10Gi of disk space available, and is less than 60% full.
```

i think what you had originally was best here with the addition of fail. i didnt mean to suggest that you apply the same as containerd

```suggestion
   - diskUsage:
        checkName: "Ephemeral Disk Usage /var/lib/rook"
        collectorName: "Ephemeral Disk Usage /var/lib/rook"
        exclude: '{{kurl not .Installer.Spec.Containerd.Version}}'
        outcomes:
          - fail:
              when: "used/total > 80%"
              message: The disk containing directory /var/lib/rook is more than 80% full
          - fail:
              when: "available < 10Gi"
              message: The disk containing directory /var/lib/rook has less than 10Gi of disk space available
          - pass:
              message: The disk containing directory /var/lib/rook has sufficient space
```